CHANGELOG.md

```diff
@@ -6,7 +6,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
```

Comment:
Please keep it

Reply:
Reverting to re-add based on comment from @xrmx